e2e/tests/0060-selinux-smoke-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Sorry, but we're obsoleting the `e2e/` tests in favor of RF. This needs to be rewritten :fearful:

Reply:
lol no worries, earlier we had discussed placing these not in a RF due to the VM concerns, but I think we're good there for now.